src/core/etl/src/Flow/ETL/PHP/Type/ArrayContentDetector.php

```diff
@@ -40,26 +40,9 @@ public function isList() : bool
         return 1 === $this->uniqueValuesType->without(ArrayType::empty(), new NullType())->count();
     }
 
-    public function isMap() : bool
```

Comment:
I think I would keep this method for now, we are not going to use it here, but it could be used by a MapEntry to validate if the given input types are valid for the map